src/context/WalletProvider/NewWalletViews/NewWalletViewsSwitch.tsx

```diff
@@ -129,15 +131,29 @@ export const NewWalletViewsSwitch = () => {
   }, [toast, translate, wallet])
 
   const handleBack = useCallback(async () => {
-    history.goBack()
+    const { pathname } = history.location
+
+    if (location.state?.vault && pathname === NativeWalletRoutes.CreateTest) {
+      history.replace({
+        pathname: NativeWalletRoutes.Create,
+        state: { vault: location.state.vault },
+      })
+
+      // Add this at the end of the callstack so we can store the vault in the state
```

Comment:
nitpick: use next tick terminology in comment, not super accurate either but *more* accurately reflect the intent

q1: is 250ms heuristics guaranteed to consistently work depending  in the main thread load?
q2: would 0ms work?

Reply:
Changed it to 0ms so it's not missleading that time is not a factor (in [d8091b3](https://github.com/shapeshift/web/pull/8885/commits/d8091b329778cd99a575de52a02ddf6598afd3ed) )